docs/UserGuide.md

```diff
@@ -351,29 +359,26 @@ Want to remove a contact? Simply use our `delete` command with the index `INDEX`
 
 #### Adding a tag : `tagadd`
 
-Adds a tag to the tag list.
+Do you want to add a tag without any contacts? Use the `tagadd` command to achieve this!
 
 Format: `tagadd t/TAG`
 
 * Creates a specified tag without the need of a contact.
 * Tag will not be created if it already exists in the tag list.
 
 Examples:
-* `tagadd t/CS2103T` creates a tag named CS2103T without assigning to any contact
-
-#### Listing all tags: `taglist`
-
-Want to see all the tags you have added? Display them with simply the `taglist` command.
+* `tagadd t/person` creates a tag named personal without any contact being assigned to the tag.
```

Comment:
i think there's a slight typo here? not really sure if it's person or personal

Reply:
oh yes thanks